### PR TITLE
added sources from A-C

### PR DESCRIPTION
**Pull request by thja**

### CODE REVIEW

**Comment by thja** [comment]
I signed it!

File: sources/active_campaign.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/active_campaign.json]: filename should match id, activecampaign.json

File: sources/active_campaign.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  "id": "ACTIVECAMPAIGN",
+  "name": "ActiveCampaign",
+  "categories": ["MARKETING"],
+  "organization": "ACTIVECAMPAIGN",
+  "iconUrl": "https://adverity-gds-static.s3-eu-west-1.amazonaws.com/icons/active_campaign.png",
+  "sourceUrl": "https://www.activecampaign.com/api/overview.php",
```
Comment (pfrisella) [inline sources/active_campaign.json]: the sourceUrl is to help users learn more about the service/source not the potential method used to fetch data. 

If this isn't a specific subset of data then I'd suggest the best link would be just https://www.activecampaign.com

File: sources/affilinet_advertiser.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "AFFILINET_ADVERTISER",
+  "name": "Affilinet Advertiser Webservices",
+  "categories": ["ADVERTISING"],
+  "organization": "AFFILINET",
```
Comment (pfrisella) [inline sources/affilinet_advertiser.json]: There is already an Affilinet source but it looks like this is a more specific subset of data, which is perfectly fine. However, the other Affilinet lists AWIN as the org, which seems more appropriate. If you agree please update and you can remove organizations/affilinet.json

File: sources/affilinet_publisher.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "AFFILINET_PUBLISHER",
+  "name": "Affilinet Publisher Webservices",
+  "categories": ["ADVERTISING"],
+  "organization": "AFFILINET",
```
Comment (pfrisella) [inline sources/affilinet_publisher.json]: There is already an Affilinet source but it looks like this is a more specific subset of data, which is perfectly fine. However, the other Affilinet lists AWIN as the org, which seems more appropriate. If you agree please update and you can remove organizations/affilinet.json

File: sources/affilinet_advertiser.json [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+  "iconUrl": "https://adverity-gds-static.s3-eu-west-1.amazonaws.com/icons/affilinet.png",
+  "sourceUrl": "http://developer.affili.net/advertiser-webservices/documentation/",
```
Comment (pfrisella) [inline sources/affilinet_advertiser.json]: is there a user facing page that users can learn more about this source?

File: sources/affilinet_publisher.json [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+  "iconUrl": "https://adverity-gds-static.s3-eu-west-1.amazonaws.com/icons/affilinet.png",
+  "sourceUrl": "http://developer.affili.net/publisher-webservices/documentation/",
```
Comment (pfrisella) [inline sources/affilinet_publisher.json]: is there a user facing page that users can learn more about this source?

File: sources/amazon_redshift.json [hunk 7]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "AWS_REDSHIFT",
```
Comment (pfrisella) [inline sources/amazon_redshift.json]: should match filename, AMAZON_REDSHIFT

File: organizations/affilinet.json [hunk 8]
```diff
@@ -0,0 +1,6 @@
+{
+  "id": "AFFILINET",
```
Comment (pfrisella) [inline organizations/affilinet.json]: See comments for Affilinet sources, you may not need this org if you use the existing AWIN org.

File: sources/amazon_s3.json [hunk 9]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "AWS_S3",
```
Comment (pfrisella) [inline sources/amazon_s3.json]: match file, AMAZON_S3

File: sources/awr.json [hunk 10]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "AWR",
```
Comment (pfrisella) [inline sources/awr.json]: for readability can you make this ADVANCED_WEB_RANKING and update filename accordingly.

File: organizations/awr.json [hunk 11]
```diff
@@ -0,0 +1,6 @@
+{
+  "id": "AWR",
```
Comment (pfrisella) [inline organizations/awr.json]: for readability can you make this ADVANCED_WEB_RANKING and update filename accordingly.

File: sources/awr.json [hunk 12]
```diff
@@ -0,0 +1,9 @@
...
+  "name": "Advanced Web Ranking",
+  "categories": ["SEO"],
+  "organization": "AWR",
```
Comment (pfrisella) [inline sources/awr.json]: update org if name is changed to ADVANCED_WEB_RANKING

File: sources/campaign_monitor.json [hunk 13]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "CAMPAIGNMONITOR",
```
Comment (pfrisella) [inline sources/campaign_monitor.json]: CAMPAIGN_MONITOR

File: sources/campaign_monitor.json [hunk 14]
```diff
@@ -0,0 +1,9 @@
...
+  "name": "Campaign Monitor",
+  "categories": ["EMAIL"],
+  "organization": "CAMPAIGNMONITOR",
```
Comment (pfrisella) [inline sources/campaign_monitor.json]: CAMPAIGN_MONITOR

File: sources/clever_reach.json [hunk 15]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/clever_reach.json]: filename should match Id, cleverreach.json

File: organizations/affilinet.json (hunk 8 above)
Comment (gernotkeuschnig) [reply organizations/affilinet.json]: yes, true

**Comment by thja** [comment]
@pfrisella please review

**Pull request merged**